test/test_ops.py

```diff
@@ -2162,6 +2162,8 @@ def map_to_fake(e):
                 ):
                     if not isinstance(fake_out, torch.Tensor):
                         self.assertTrue(not isinstance(real_out, torch.Tensor))
+                        if isinstance(fake_out, (bool, int, float)):
```

Comment:
Maybe leave a comment here, my interpretation of this code is that, previously  "if the fake out is not a tensor, make sure the real out isn't a tensor, everything else we don't care about' but now we do care but only about certain things, why these things?

Reply:
Switched it to check everything